### PR TITLE
Add support for lttng

### PR DESCRIPTION
**Pull request by aholza**
In order to support builds against lttng, lets add some packageconfig option.

### CODE REVIEW

**Comment by milianw** [comment]
this isn't enough, we also need to patch the native side of things, afaik via:

```
qtbase-native_git.bb:PACKAGECONFIG[lttng] = "-trace lttng,-no-trace,"
nativesdk-qtbase_git.bb:PACKAGECONFIG[lttng] = "-trace lttng,-no-trace,"
```

**Comment by milianw** [comment]
or not, based on: https://codereview.qt-project.org/c/qt/qtbase/+/276857 then tracegen will always be compiled

File: recipes-qt/qt5/qtbase_git.bb [hunk 1]
```diff
@@ -119,6 +119,7 @@ PACKAGECONFIG[xkb] = "-xkb,-no-xkb -no-xkbcommon,libxkbcommon"
 PACKAGECONFIG[xkbcommon] = "-xkbcommon,-no-xkbcommon,libxkbcommon,xkeyboard-config"
 PACKAGECONFIG[evdev] = "-evdev,-no-evdev"
 PACKAGECONFIG[mtdev] = "-mtdev,-no-mtdev,mtdev"
+PACKAGECONFIG[lttng] = "-trace lttng,,lttng-ust"
```
Comment (shr-project) [inline recipes-qt/qt5/qtbase_git.bb]: I'll add "-trace no" to explicitly disable it when not enabled.

That's what we're using:
https://github.com/webosose/meta-webosose/blob/master/meta-webos/recipes-qt/qt5/qtbase_git.bbappend#L53

**Comment by shr-project** [comment]
In zeus-next and master-next now.

Please add your Signed-off-by tag as well.

**Pull request closed without merging**